Test/NativeUnitTest/NativeUnitTest.cpp

```diff
@@ -12,12 +12,9 @@ namespace NativeUnitTestForMain
 		
 		TEST_METHOD(TestIfPathExists)
 		{
-			int charOccurs[possibleChars];
-			for (int i = 0; i < possibleChars; ++i) {
-				charOccurs[i] = 0;
-			}
-			Assert::IsFalse(calculateOccursInFile("randompath.txt", {}));
-			Assert::IsTrue(calculateOccursInFile("D:/bruhmoment/Abyss/ToDo.txt", charOccurs));
+			int charOccurs[possibleChars] = {};
+			std::string path = "D:/bruhmoment/Abyss/ToDo.txt";
```

Comment:
Do this file exist on my PC when I run this test? Of course - NO
You can create temp (in system temp folder, which is unique for each system) file for the test, do the test and delete it.

Reply:
Will change in a later PR, just so I don't have to jump back and forth between them